integration_tests/snapshots/logs/async-metrics_node10.log

```diff
@@ -3,15 +3,15 @@ START RequestId: XXXX Version: $LATEST
 {"e":XXXX,"m":"serverless.integration_test.execution","t":["tagkey:tagvalue","eventsource:APIGateway","dd_lambda_layer:datadog-nodev10.XX.X"],"v":1}
```

Comment:
We should exclude snapshots from the whitespace checks, since this whitespace is technically important

Reply:
Good point!